### PR TITLE
Feature/requested ux 1

### PR DESCRIPTION
**Pull request by tonyantony300**

### CODE REVIEW

File: src/lib/components/dashboard/components/Placeholder.svelte [hunk 1]
```diff
@@ -19,24 +22,62 @@
     copiedToClipboard = true;
     isLoaderActive = false;
   };
+
+  onMount(async () => {
+    const user = await getUser();
```
Comment (AbrahamGeorge8547) [inline src/lib/components/dashboard/components/Placeholder.svelte]: please use the user saved in local storage

File: src/lib/components/dashboard/apis.ts [hunk 2]
```diff
@@ -19,6 +19,6 @@ export {
     fetchUsersWithoutGroupAccess, removeGroupFromFolder, removeGroupFromCredential, removeUserFromGroup, removeGroup
 } from '../../apis/group.api';
 export {
-    fetchSignedUpUsers, createUser, fetchCredentialUsers, removeUserFromFolder,
-    removeUserFromCredential, deleteUser, checkUserNameExists, fetchAllUsers
+    fetchAllUsers, createUser, fetchCredentialUsers, removeUserFromFolder,
```
Comment (AbrahamGeorge8547) [inline src/lib/components/dashboard/apis.ts]: fetchSignedUpUsers removed?

File: src/lib/components/popup/Home.svelte [hunk 3]
```diff
@@ -30,15 +30,20 @@
...
     });
     const activeTab = tabs[0];
     if (activeTab && activeTab.url) domain = new URL(activeTab.url).hostname;
+    return domain;
+  };
+
+  const fetchCredentialsOfCurrentDomin = async () => {
+    const responseJson = await fetchAllUserUrls();
+    const urls = responseJson.data;
+    await readDomain();
```
Comment (AbrahamGeorge8547) [inline src/lib/components/popup/Home.svelte]: domain is not assigned

File: src/lib/components/dashboard/components/Placeholder.svelte (hunk 1 above)
Comment (tonyantony300) [reply src/lib/components/dashboard/components/Placeholder.svelte]: done

**Review by AbrahamGeorge8547 — APPROVED** [review]

**Pull request merged**